src/ansible/roles/keycloak/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+keycloak_name: "keycloak"
+#keycloak_version: "19.0.3"
+keycloak_version: "17.0.1"
```

Comment:
Why version `17` when upstream released version `21` two months ago? I'm not following keycloak releases profoundly but I'd expect to use the latest version (`21`) or the previous one (`20`)

Reply:
17 and 19 are just the versions that we've been testing the new SCIM user storage plugin with.   I'll look into locking it down to a newer version but, we will need to manage that versioning somehow so we can know which version of KC is in the container.  Thoughts?

Reply:
I think it's fine to keep the reference to the keycloak version in use, but as mentioned previously I'd expect it to be one of the newest ones.

Reply:
OK, I set the var to the current latest.  removing commented ones in the upcoming push